ERCS/erc-7526.md

```diff
@@ -0,0 +1,128 @@
+---
+eip: 7526
+title: On-chain NFT royalty enforcement
+description: A game-theoretic mechanism to induce truthful revelation of NFT sales and their value to allow for decentralised royalty enforcement.
```

Comment:
```suggestion
description: An interface for game-theoretic mechanism to induce truthful revelation of NFT sales and their value to allow for decentralised royalty enforcement.
```


ERCs are standard track: most ERCs are used to propose a way for interoperability. Using it to propose a "mechanism" is a bit confusing. Maybe add "interface" in your summary.

Also, I don't see game theory being used other than the motivation. Can you show it in the reference implementation of what you refer to?